dev/run-tests.py

```diff
@@ -33,6 +33,9 @@
 from sparktestsupport.toposort import toposort_flatten
 import sparktestsupport.modules as modules
 
+always_excluded_tags = [
```

Comment:
`always_excluded_tags` might be misleading because this should be tested in the proper environments.